webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -123,7 +119,10 @@ const ModalForm: React.FC<Props> = ({ form, user, viewOnly, roles }) => {
             </Select>
           </Form.Item>
           <Typography.Text type="secondary">
-            Note that roles inherited from user groups or workspaces cannot be removed here.
+            Groups may have additional inherited workspace roles not reflected here. &nbsp;
```

Comment:
I think this one should say "Users" and "global or workspace roles"?

Reply:
Thanks!